sympy/functions/elementary/trigonometric.py

```diff
@@ -2240,8 +2272,8 @@ def _eval_is_real(self):
         return self.args[0].is_real
 
     def _eval_rewrite_as_log(self, x):
-        return S.ImaginaryUnit/2 * (log(
-            (S(1) - S.ImaginaryUnit * x)/(S(1) + S.ImaginaryUnit * x)))
+        return S.ImaginaryUnit/2 * (log(S(1) - S.ImaginaryUnit * x)
+            - log(S(1) + S.ImaginaryUnit * x))
```

Comment:
@smichr Because it's complex logarithm, two forms can differ by multiples of pi.
i.e., `assert atan(5*I).evalf() == atan(x).rewrite(log).evalf(subs={x:5*I})` fails with previous form.